### DISCUSSION TITLE
2024-Inthon-Datathon/notice

### DISCUSSION THREAD

**Discussion by giscus** [KU-InThon-Datathon/2024-Inthon-Datathon#10] [Announcements]
# 2024-Inthon-Datathon/notice

Datathon Track

https://ku-inthon-datathon.github.io/2024-Inthon-Datathon/notice.html

**Comment by KyleBae1017** [comment]
Baseline Code에 오류가 발생하여 수정하셔야 하는 셀이 있습니다. 관련 내용은 QnA 탭 및 카카오톡 채팅방을 참조해주세요. 감사합니다.

There is a cell in the Baseline Code that needs correction due to an error. Please refer to the Q&A tab and KakaoTalk Chat Room for more information. Thank you.

**Comment by yms020615** [comment]
Reliability 점수 채점을 위한 사이트가 3시에서 4시 사이에 배포될 예정입니다.
예정대로 제출은 3시부터 가능합니다.
생성해 놓으신 캡션에 대한 Reliability 점수 평가를 원하시는 조는 아래 메일로 제출물을 첨부하여 전송해 주세요.

cooki0615@korea.ac.kr

**Reply by yms020615** [reply]
카카오톡(윤민서)으로도 제출 가능합니다.
메일이나 카카오톡을 통한 제출 또한 전체 제출 횟수에 포함되는 점 유의 바랍니다.
Submission 형식에 맞지 않아 채점 코드에서 오류가 나는 제출물은 전체 제출 횟수에 포함되지 않습니다.

**Reply by yms020615** [reply]
The site for scoring Reliability will be released between 3 and 4 PM. Submissions can be made from 3 PM as scheduled.

For teams wishing to evaluate the Reliability scores of their generated captions, please attach and send your submission to the email below:

cooki0615@korea.ac.kr

Submissions can also be sent via KakaoTalk (윤민서). Please note that submissions sent via email or KakaoTalk will be counted toward the total number of submissions. Submissions that do not match the required format and cause errors in the scoring code will not be counted toward the total submission count.

**Comment by KyleBae1017** [comment]
Submission 웹사이트에 사용될 계정을 팀 당 하나씩 저희가 생성하여 제공해드리려고 합니다.
각 팀 별로 계정의 ID로 사용할 팀 이름을 자유롭게 정하여 팀원분들의 성함과 함께 이 글에 댓글로 달아주세요. 감사합니다.

We will create and provide one account per team for the Submission website. 
Please choose a team name to use as the account ID for each team and post it in the comments section of this post, along with the names of your team members. Thank you.

**Reply by KyleBae1017** [reply]
Team 1. 김창빈 박정우 이제윤 염재원

**Reply by strn18** [reply]
5팀: 반Ban / 문정민 김윤서 안효원 전현준

**Reply by wlsguur** [reply]
team9
최진혁 김지연 조윤지 김명진

**Reply by BaconTomatoDeluxe** [reply]
에잇
정진욱 이창엽 전하은 권서은

**Reply by alepnawi** [reply]
10팀: nasilemak
파와즈 마히라 줄리아 알리프

**Reply by KyleBae1017** [reply]
팀투
김태관 최승민 김태우 강지융

**Reply by es2ilver** [reply]
라스트댄스
이정은 김민경 김은진 허준환

**Reply by Soeayun** [reply]
3팀: 트러플스타
남동연 안동현 신민서 허주원

**Reply by yws0322** [reply]
4ward
문승기 김재영 김유진 서연우

**Reply by KyleBae1017** [reply]
6조: 스모어
이성민 김민준 윤다현 송재헌

**Comment by yms020615** [comment]
Submission 결과물을 채점하는 데 수십 분이 소요될 수 있습니다. 이 점 참고 바랍니다.

Please note that it may take tens of minutes to grade the submission results. Thank you for your patience.

**Comment by KyleBae1017** [comment]
16시 50분 기준 리더보드를 카카오톡에 공개하였습니다. 채점 서버 배포가 늦어지고 있는 점 양해 부탁드리며, 제출은 추가적인 공지가 있기 전까지 카카오톡 혹은 메일로 부탁드립니다. 죄송합니다. 2팀은 채점을 진행중이니 참고 바랍니다.

**Reply by KyleBae1017** [reply]
As of 4:50 PM, the leaderboard has been shared on KakaoTalk. We apologize for the delay in deploying the grading server, and we kindly ask that submissions be made via KakaoTalk or email until further notice. Our apologies for the inconvenience. Please note that grading is currently in progress for Team 2.

**Comment by yms020615** [comment]
transformers의 `generate` 메소드에는 **repetition_penalty**라는 하이퍼파라미터가 있습니다.

현재 비교적 정상적인 caption을 채점하는 데는 약 10~20분 정도가 소요되나, 제출물에 동어 반복이 지나치게 많을 때에는 1시간을 넘어서는 경우도 발생하고 있습니다.

제출하기 전에 여러분의 qualitative result를 확인을 하고 제출을 하시면 여러분의 점수를 더욱 빨리 알 수 있고 제출 횟수도 절약할 수 있습니다.

감사합니다.


In the `generate` method of Transformers, there is a hyperparameter called **repetition_penalty**.

Currently, it takes about 10-20 minutes to evaluate a relatively normal caption, but when there is excessive repetition in the submission, it can take over an hour.

Please review your qualitative results before submitting to receive your score faster and save on the number of submissions.

Thank you.

**Comment by yms020615** [comment]
Reliability 점수 재현을 위하여 컴피티션 종료 이후에 private score가 제일 높은 코드를 제출해야 하니, 코드를 미리 버전별로 정리해 두시는 것을 권장합니다.


To reproduce the Reliability score, you will need to submit the code with the highest private score after the competition ends, so it is recommended to organize your code versions in advance.

**Comment by KyleBae1017** [comment]
채점에 일정 시간이 소요되기 때문에 제출이 지나치게 몰릴 경우 리더보드 업데이트가 지연될 수 있는 상황입니다. 최종 마감 후 최대한 빠르게 Private 리더보드를 확정해야하므로 이를 위해 **각 팀 별로 오늘 12시에서 13시 사이의 제출은 최대 1회로 제한하고자 합니다.** 감사합니다.

Since grading takes a certain amount of time, there may be delays in leaderboard updates if submissions become excessively concentrated. To finalize the private leaderboard as quickly as possible after the final deadline, **each team will be limited to a maximum of one submission between 12:00 and 1:00 PM today**. Thank you for your understanding.

**Comment by KyleBae1017** [comment]
지금부터 csv 파일 제출은 윤민서 TF님이 아닌 저에게 카카오톡으로 해주세요. 감사합니다.

From now on, the submission (of csv file) is to "배민성", not to "윤민서" via KakaoTalk. Thank you.
